lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -1,138 +1,232 @@
+                            </button>
+                        </div>
+                        <div *ngIf="isEditingTags" class="adf-metadata-action-buttons">
+                            <button mat-icon-button
+                                    [attr.title]="'CORE.METADATA.ACTIONS.CANCEL' | translate"
+                                    (click)="onCancelTagsEdit($event)"
+                                    data-automation-id="reset-tags-metadata"
+                                    class="adf-metadata-action-buttons-clear">
+                                <mat-icon>clear</mat-icon>
+                            </button>
+                            <button mat-icon-button
+                                    [attr.title]="'CORE.METADATA.ACTIONS.SAVE' | translate"
+                                    (click)="onSaveTagsChanges($event)"
+                                    color="primary"
+                                    data-automation-id="save-tags-metadata"
+                                    [disabled]="!hasMetadataChanged">
+                                <mat-icon>check</mat-icon>
+                            </button>
+                        </div>
+                        </adf-content-metadata-header>
                 </mat-expansion-panel-header>
-                <p *ngFor="let tag of tags" class="adf-metadata-properties-tag">{{ tag }}</p>
-            </mat-expansion-panel>
-            <div
-                *ngIf="editable"
-                class="adf-metadata-properties-tags">
-                <div class="adf-metadata-properties-tags-title">
-                    <p>{{ 'METADATA.BASIC.TAGS' | translate }}</p>
-                    <button
-                        data-automation-id="showing-tag-input-button"
-                        mat-icon-button
-                        [attr.title]="'METADATA.BASIC.ADD_TAG_TOOLTIP' | translate"
-                        (click)="tagNameControlVisible = true"
-                        [hidden]="tagNameControlVisible || saving">
-                        <mat-icon>add</mat-icon>
-                    </button>
+                <div *ngIf="!isEditingTags" class="adf-metadata-properties-tags">
+                    <span *ngFor="let tag of tags" class="adf-metadata-properties-tag">{{ tag }}</span>
                 </div>
-                <adf-tags-creator
-                    [(tagNameControlVisible)]="tagNameControlVisible"
-                    (tagsChange)="storeTagsToAssign($event)"
-                    [mode]="tagsCreatorMode"
-                    [tags]="assignedTags"
-                    [disabledTagsRemoving]="saving">
-                </adf-tags-creator>
-            </div>
+                <div *ngIf="showEmptyTagMessage()" class="adf-metadata-no-item-added">
+                    {{ 'METADATA.BASIC.NO_TAGS_ADDED' | translate }}
+                </div>
+                <div *ngIf="isEditingTags" class="adf-metadata-properties-tags">
```

Comment:
same here - div seems redundant - we can apply class and *ngIf to adf-tags-creator directly